pgmpy/base/DirectedGraph.py

```diff
@@ -177,12 +171,11 @@ def moralize(self):
 
         Examples
         --------
-        >>> from pgmpy.base import DirectedGraph
         >>> G = DirectedGraph([('diff', 'grade'), ('intel', 'grade')])
         >>> moral_graph = G.moralize()
-        >>> moral_graph.edges()
+        >>> moral_graph.edges() # doctest: +IGNORE_RESULT
```

Comment:
But adding `sort` there will confuse the user, and furthermore we will have to do it anyways for results having object address, like
`<Factor representing phi(x1:2, x2:2, x3:2) at 0x7f8188fcaa90>`
Because they are always going to be variable.